.github/workflows/bigbro-prod-cd.yml

```
@@ -16,14 +16,12 @@ jobs:
+          host: ${{ secrets.SSH_HOST_PROD }}
+          port: ${{ secrets.SSH_PORT_PROD }}
+          username: ${{ secrets.SSH_USERNAME_PROD }}
+          key: ${{ secrets.SSH_PRIVATE_KEY_PROD }}
           script: |
             set -e
             cd bigbro_company_server
             git pull origin main
-            docker-compose -f infra/compose/docker-compose.yml stop prod-web
-            docker system prune -af --volumes=false
-            docker-compose -f infra/compose/docker-compose.yml up -d --build prod-web
+            ./deploy.sh
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)